Multichain-proposal.md

```diff
@@ -0,0 +1,112 @@
+# Multichain Proposal for Cardano
```

Comment:
Flow diagrams of each use case would be great to fully understand the proposal.

Reply:
Please check this flow diagram with comments. Thanks
[Cardano Flow.pdf](https://github.com/input-output-hk/Bridges-working-group/files/9525098/Cardano.Flow.pdf)

Reply:
It would be much better to have flow diagrams corresponding with each use case, showing all the actions needed on the Cardano side:

* Case 1: Send Cardano native token to Ethereum.
* Case 2: Send back a wrapped token from Ethereum to Cardano.
* Case 3: Send Ethereum token to Cardano.
* Case 4: Send back a wrapped token from Cardano to Ethereum.

For each case specify what is the process if it's the first time a token is bridged (it will imply defining a new minting policy)